native/packages/react-native-bpk-component-button-link/src/BpkButtonLink.android.js

```diff
@@ -75,13 +78,21 @@ const BpkButtonLink = (props: Props) => {
     containerStyle.push(style);
   }
 
+  // Disabled styles must be applied last to ensure they
```

Comment:
Remove this comment, the snapshot tests catch this.

Reply:
Done!

Reply:
Actually I'd argue that userland styles come last. One of the key lessons we've learned in the past is that if consumers want to customise stuff they'll either find a way or just not use the component (that latter of which is entirely undesirable from a design system perspective).

Reply:
Most other components follow this order of precedence...

Reply:
Yeah that's fair, I'll move it.

Reply:
Done!